packages/modular-scripts/react-scripts/config/modules.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const paths = require('./paths');
+const chalk = require('react-dev-utils/chalk');
```

Comment:
We should decide which chalk package we want to use throughout the project. Do you want to prune out unnecessary dependencies in this PR or a separate smaller one?

Reply:
I think pruning should come separately - I want to try and keep the changes as small as possible.